components/brave_wallet_ui/common/hooks/use-scroll-into-view.tsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+import * as React from 'react'
+
+export function useScrollIntoView () {
+  // Prevents scrolling into view again if re-render occurs
+  let ignore = false
+  const scrollIntoView = React.useCallback((ref: HTMLButtonElement | HTMLDivElement | null) => {
```

Comment:
Can we replace `ref: HTMLButtonElement | HTMLDivElement | null` with `ref: HTMLElement | null` to allow more element types?

Reply:
Updated!